cypress/integration/strava.spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+context('<Strava />', () => {
+  it('it loads strava activity with header, footer and optional content', () => {
+    cy.visit('/iframe.html?id=components-strava--usage&viewMode=story');
+
+    // Make sure the activity is there
+    cy.getIframeBody().find('#activity').should('exist');
+
+    // The activity should have a header with an avatar
+    cy.getIframeBody().find('.header').should('exist');
+    cy.getIframeBody().find('.header').find('.avatar').should('exist');
+
+
+    // The activity should have a content
+    /* Info
+    Not all activities have actual content (i.e. a map or photos),
+    so the .content div might actually be empty, but it will still be there.
+    For example, indoor running will have no content.
+    Some activities have maps and some have photos, so it's not reliable to check for either
+    using this setup with storybook running.
+    */
+   cy.getIframeBody().find('.content').should('exist');
```

Comment:
This is a fair point. I think for now this is totally fine. our "sprint goal" if you like was just to have integration tests in place to ensure the provider scripts actually loads. What it loads based on props that configure it will likely come later. On pretty much all components we need to come up with a url constructor method unique to each component which can then be tested. Once that's done we can probably look to dig in a little more with E2E tests. Great job on this though... Thanks

Reply:
Yep, that's great! I thought we'd benefit from having this in there at some point - I thought a comment makes sense for the next person who might work on it. 👍 

Thanks for taking a look so quickly!